metrics/chronos_httpserver.nim

```diff
@@ -65,7 +65,10 @@ when defined(metrics):
           printError(exc.msg)
 
     let socketFlags = {ServerFlags.ReuseAddr}
-    let res = HttpServerRef.new(address, cb, socketFlags = socketFlags)
+    let res = try: HttpServerRef.new(address, cb, socketFlags = socketFlags)
```

Comment:
`new` procedure do not produce `CatchableError` anymore, i have workaround Nim's issue in https://github.com/status-im/nim-chronos/commit/f2e4d447d6aec99b3641d51994650769c5c00d02